spec/swagger_spec.rb

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Extra newline?

Reply:
:+1: removed, deleted a byebug require but forgot the extra line